data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+insert into animals values (default, 'Agumon', '20200203', 0, true, 10.23, CURRENT_USER, now()); 
+insert into animals values (default, 'Gabumon', '20181115', 2, true, 8.00, CURRENT_USER, now());
+insert into animals values (default, 'Pikachu', '20210107', 1, false, 15.04, CURRENT_USER, now());
+insert into animals values (default, 'Devimon', '20170512', 5, true, 11.00, CURRENT_USER, now());
```

Comment:
- [ ] The insertion way wouldn't be like this, you need to specify the column names you need to insert with the corresponding values.
Sample:  `INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23'); `

Reply:
Hi Emmy, kindly I would like to request you to change your review. As you know, in SQL if you use the right sequence of values, isn't necessary to inform the fields on this insert. It's a normal procedure in the "real world" of IT. You'll see a lot of this. And please kindly check the code before demanding a change request, because when you run it, all will work perfectly. Just for your information, I'm certified in Oracle and DB2 databases. 
Can you change your review or do I need to open a second opinion for this lack of knowledge?